### ISSUE TITLE
Allow the Application Menu to appear just by using a mouse without needing a keyboard or a trackpad.

### ISSUE THREAD

**Issue by Mesum-Hussain** [pop-os/gnome-shell-extension-pop-cosmic#327]
In my opinion the whole point of an app menu is to let the users launch the apps using the mouse. If I need to click a key binding on my keyboard to make it appear then use the mouse to select the app, then in my opinion its redundant because we have a better launcher for keyboard oriented workflow already and if I have to use keyboard then I would rather use that. The app menu should allow the apps to be launch just by using the mouse in my opinion. It's better in case of a trackpad as we can summon it with the gesture, but if you are on a desktop computer then you are out of luck. This problem didn't existed on Pop OS 21.04 as the app menu could have been summoned by dragging the mouse from the left side of the screen (a little inconvenient still, but did the job).

**Comment by leviport** [comment]
The applications library can be opened both by clicking "Applications" in the left side of the top bar, and by clicking the "Show Applications" icons icon on the dock. Do those fit the bill?

**Comment by Mesum-Hussain** [comment]
> The applications library can be opened both by clicking "Applications" in the left side of the top bar, and by clicking the "Show Applications" icons icon on the dock. Do those fit the bill?

Sorry my bad. Actually I have both of them disabled on my PC for a long enough time (majorly for getting a minimalist look) that I completely forgot those options exists. Although with the previous app menu I didn't miss not having those two enabled because the app menu still was accessible by a little drag of the mouse from the left side of the screen, which the new app menu doesn't do. I guess I should enable the applications button in the top bar to solve this. Though now that I think of it, another reason for which I disabled it was that I have Hot Corner for Workspaces enabled and many times it accidentally got triggered when I tried to click the Applications button. It was causing a little inconvenience and also getting rid of the button gave a much cleaner look, so I got rid of it. But even enabling that won't work in my setup because I have an extension which shows the top bar only in Workspace Overview and hides rest of the time, but that's not an issue with Pop OS, it's my personal problem. I guess I have to pick my poison.....can't get all the features at the same time.
Sorry for taking your time.
